### ISSUE TITLE
Addresses with multiple units don't always work with trash day intent

### ISSUE THREAD

**Issue by jmartini** [codeforboston/voiceapp311#244]
@jmartini I tested with a bunch of addresses, most worked fine. I had issues with `66 Seaverns Avenue` in JP, and `1754 Commonwealth Ave` in Brighton (screenshot below).

![Screenshot 2019-05-28 22 57 45](https://user-images.githubusercontent.com/7052789/58526181-265cad80-819c-11e9-828c-f2d1b251233b.png)

_Originally posted by @thiteixeira in https://github.com/codeforboston/voiceapp311/pull/243#issuecomment-496765698_

**Comment by thiteixeira** [comment]
Can we do something like this and select the first address?

```
#!/usr/bin/env python
from pprint import pprint
import requests
import json

address = '1754 Commonwealth Ave'

base_url = 'https://api.recollect.net/api/areas/Boston/services/310/address-suggest?'
query = 'q=' + '+'.join(address.split())
locale = '&locale=en-US'
_ = '&_=1559135785593'

url = base_url + query + locale + _

res = requests.get(url)
if res.status_code != 200:
  print('Error: ' + res.status_code)
else:
  pprint(json.loads(res.text))
```

**Comment by jmartini** [comment]
The problem is when there are more then one addresses in different neighborhoods, so the first response might not actually be the user's address. See #100 for the original issue why we added zip code verification in these cases

**Comment by jmartini** [comment]
Closed by #245
